src/client/dfs/obj.c

```diff
@@ -1013,6 +1022,9 @@ statx_task(tse_task_t *task)
 err1_out:
 	D_FREE(op_args);
 	daos_obj_close(args->parent_oh, NULL);
+
+	if (rc == 0)
+		DFS_OP_STAT_INCR(args->dfs, DOS_OSTATX);
```

Comment:
i would not make a statx, just stat is fine in this case. statx is just an async version of the API

Reply:
So that comment would probably apply to the other -X calls? I was wondering about that... On the one hand, I can see the simplicity of consolidating them. On the other, I wonder if it's useful to see when a client is using the async versions (performance improvements?). I'll defer to your guidance on that.

Reply:
yea i think users care less about the type of the call vs the actual metrics of what is being done. so yes i agree consolidating them with removing the X would be better and would avoid confusion for users between 2 different but really the same metric

Reply:
Done, thanks.